server/test/controller/lick.spec.ts

```diff
@@ -28,12 +26,116 @@ describe('Unit test: Lick endpoint', () => {
         sandbox.restore()
     })
 
+    /**
+     * Test createLick()
+     */
+    it('should CREATE lick with valid data', async () => {
+        const lickOwner: User = new User()
+        lickOwner.id = 1 // must set id
+
+        const now: Date = new Date();
+        sandbox.useFakeTimers(now.getTime()); // stub new Date() to return now
+
+        // This is what the endpoint will store in the database / is returned
+        const fakeLick: Lick = new Lick()
+        fakeLick.name = "cool lick";
+        fakeLick.description = "good one";
+        fakeLick.dateUploaded = now;
+        fakeLick.tab = "";
+        fakeLick.tuning = "drop d";
```

Comment:
After looking at the tests that use tuning like this, I think it might be a good idea to have tuning as an Enum for safety. If it's predefined as an enum it also provides a good basis for what the frontend should have as options for the user to select.

However, this doesn't have to be addressed in this ticket.

Reply:
We could make it an enum, I was thinking the best way to do it would be to pass a string of chars to the server like EADGBE which the server could parse so the user could do any tuning they want, however for this prototype, it might be best just to use an enum with some common options